src/backend/Dockerfile

```diff
@@ -98,6 +98,14 @@ COPY --from=base-with-deps-dev /venv /venv
 # ENV DB_HOST=postgresql \
 #     DB_PORT=5432
 
+# Switch to root to install packages
+USER root
+RUN apt-get update && apt-get install -y --no-install-recommends curl \
```

Comment:
we could also use `python -c`, see https://github.com/dart-lang/dart-docker/issues/58#issuecomment-3223683013

Reply:
Didn't think of that, nice trick

Reply:
Done in https://github.com/suitenumerique/messages/pull/324/commits/a7c1d7d1ffc86e05168addea6e9eb4288e4dc4dd